### ISSUE TITLE
How to train ViT from scratch on Imagenet1k? 

### ISSUE THREAD

**Issue by ssuncheol** [lukemelas/PyTorch-Pretrained-ViT#22]
I trying to train 'ViT-B-16' , but I think that it's very hard to train ViT from scratch on Imagenet1k.
The parameters used in the experiment are as follows. 

- batch size : 1024 
- optimizer : Adamw 
- weight decay : 0.3 
- learning rate : 0.001 
- cosine warmup : 40k
